### PR TITLE
Allow to provide release notes, do not commit versions changes

### PR DESCRIPTION
**Pull request by asaarilahti**
- Allow to provide release notes (probably just a link to the GH milestone)
- Do not commit version numbers from release script
- Use patchPluginXml to set the version number instead of sed

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -28,17 +28,10 @@ if [[ -z "${JETBRAINS_HUB_TOKEN:-}" ]]
...
 
-NEW_VERSION=$1
-
-if [[ "$(uname)" == "Darwin" ]]
+if [[ "$(git branch --show-current)" != "master" ]]
   then
-    SED_ARG=(-i "")
-else
-    SED_ARG=(-i)
+    echo "Please use master branch"
```
Comment (cberg-zalando) [inline release.sh]: Should we move this to a variable to avoid DRY here? I mean, the name of the main/master branch?

Comment (asaarilahti) [reply release.sh]: Yep, makes sense. https://github.com/zalando/intellij-swagger/pull/355/commits/7a92623525c2e97bb957e563233d235863f359b5

File: release.sh [hunk 2]
```diff
@@ -28,10 +30,10 @@ if [[ -z "${JETBRAINS_HUB_TOKEN:-}" ]]
...
-if [[ "$(git branch --show-current)" != "master" ]]
+if [[ "$(git branch --show-current)" != "${DEFAULT_BRANCH}" ]]
   then
-    echo "Please use master branch"
+    echo "Please use ${DEFAULT_BRANCH} branch"
     exit 1
 fi
 
-./gradlew publishPlugin -Pversion="${1}" -PjetbrainsReleaseChannel="${2}" -PchangeNotes="${3}" -x buildSearchableOptions
+#./gradlew publishPlugin -Pversion="${1}" -PjetbrainsReleaseChannel="${2}" -PchangeNotes="${3}" -x buildSearchableOptions
```
Comment (cberg-zalando) [inline release.sh]: Was this change intended?

Comment (asaarilahti) [reply release.sh]: No, removed. Thanks for spotting!

**Review by cberg-zalando — APPROVED** [review]
